### DISCUSSION TITLE
How to benefit from metacopy, native Overlay Diff, d_type?

### DISCUSSION THREAD

**Discussion by StarpTech** [containers/podman#15515] [General]
Hi, what kernel version or configuration is necessary to benefit from all features inside a container? I researched a bit, but I couldn't figure it out. I tried Fedora 38 container image with Podman 4.2 with the Kernel `5.19.0-76051900-generic` but `podman info` always prints

```
  graphStatus:
    Backing Filesystem: overlayfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
```

On my host with Podman 3.4.4 and the same kernel `Linux pop-os 5.19.0-76051900-generic`, all features are enabled except `metacopy`.

**Comment by rhatdan** [comment]
@giuseppe PTAL
